src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1213,9 +1220,8 @@ def _first_db_span(self) -> Optional[Span]:
     def _fingerprint(self, db_hash) -> str:
         # TODO: Add more information to the hash. Since issues aren't being
         # detected yet, this doesn't matter.
-        problem_class = GroupType.PERFORMANCE_M_N_PLUS_ONE_DB_QUERIES
         full_fingerprint = hashlib.sha1(db_hash.encode("utf8")).hexdigest()
-        return f"1-{problem_class}-{full_fingerprint}"
+        return f"1-{PerformanceMNPlusOneDBQueriesGroupType.type_id}-{full_fingerprint}"
```

Comment:
Note that this was also incorrectly using the class name rather than the value like the N+1 issues, but this isn't in prod yet so I am correcting it.